JustSaying.Sample.Restaurant.OrderingApi/Program.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Design", "CA1052:Static holder types should be Static or NotInheritable")]
```

Comment:
I'd say yes by default, but you could turn off this specific one by adding `<NoWarn>$(NoWarn);CA1052</NoWarn>` to the `.csproj` file.

Reply:
Actually, this is easy to fix - just make it static.

Reply:
ahh - ta - done